rust_snuba/src/strategies/clickhouse/mod.rs

```diff
@@ -141,90 +137,3 @@ impl ProcessingStrategy<BytesInsertBatch> for ClickhouseWriterStep {
         self.inner.join(timeout)
     }
 }
-
-#[derive(Clone)]
```

Comment:
Do we need a test that actually writes a batch to clickhouse?

Reply:
right now those tests get in the way IMO